api.bs

```diff
@@ -474,7 +474,7 @@ navigator.privateAttribution.saveImpression({
 <xmp class=idl>
 dictionary PrivateAttributionImpressionOptions {
   required unsigned long histogramIndex;
-  required unsigned long filterData;
+  unsigned long filterData;
```

Comment:
Should we be setting a default or assuming that this ends up as a nullable attribute in any database?  Serious question, because `Option<u64>` and `u64` is significant.